<ui version="4.0" >
 <class>EditDrumsetBase</class>
 <widget class="QDialog" name="EditDrumsetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MuseScore: Edit Drumset</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QTreeWidget" name="pitchList" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="indentation" >
      <number>0</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Pitch</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Note</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Shortcut</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <item>
      <property name="text" >
       <string/>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="text" >
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <layout class="QFormLayout" name="formLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="name" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Note Head:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="noteHead" />
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QFormLayout" name="formLayout_2" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Staff Line:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="staffLine" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Stem Direction:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="stemDirection" >
       <item>
        <property name="text" >
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Up</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Down</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Default Voice:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="voice" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Shortcut:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QComboBox" name="shortcut" >
       <item>
        <property name="text" >
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>D</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>E</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>F</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>G</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>--</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2" >
    <widget class="Palette" native="1" name="drumNote" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Palette</class>
   <extends>QWidget</extends>
   <header>palette.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDrumsetBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDrumsetBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
